iengine/iengine-app/src/main/java/io/tapdata/flow/engine/V2/node/hazelcast/data/pdk/HazelcastTargetPdkShareCDCNode.java

```diff
@@ -466,12 +466,13 @@ private HazelcastConstruct<Document> getConstruct(String tableName, String conne
 		return constructMap.computeIfAbsent(tableName, k -> {
 			String taskId = processorBaseContext.getTaskDto().getId().toHexString();
 			String[] split = tableName.split("\\.");
-			String sign = ShareCdcTableMappingDto.genSign(taskId, connectionId, split[split.length - 1]);
+			String sign = ShareCdcTableMappingDto.genSign(connectionId, split[split.length - 1]);
```

Comment:
here has a bug, I'm not sure whether we should solve it now, mongodb allow table name has dot in it, such as: a.b, b.b, this will make data error

Reply:
I create a issue: https://tapdata.coding.net/p/tapdata/bug-tracking/issues/149111/detail